features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/checker/sql/insert/EncryptInsertSelectSupportedCheckerTest.java

```diff
@@ -38,7 +40,7 @@ class EncryptInsertSelectSupportedCheckerTest {
     @Test
     void assertIsCheck() {
         InsertStatementContext sqlStatementContext = mock(InsertStatementContext.class, RETURNS_DEEP_STUBS);
-        when(sqlStatementContext.getSqlStatement().getInsertSelect().isPresent()).thenReturn(true);
+        when(sqlStatementContext.getSqlStatement().getInsertSelect()).thenReturn(Optional.of(mock(SubquerySegment.class)));
```

Comment:
- Why does updating some third-party dependencies cause a large number of unit tests to change? Where is the original issue?

Reply:
This involves a problem. The new version of mockito does not support JDK8, so it makes no sense to do so.